sox.features.info.json

```diff
@@ -114,11 +114,11 @@
             "exclude": "SE1.0,*://*/questions/ask,*://*/questions/tagged*,*://*/questions/greatest-hits*"
         }, {
             "name": "standOutDupeCloseMigrated",
-            "desc": "Add highlighted tags to closed/on hold/duplicate/migrated questions on question lists",
+            "desc": "Add highlighted tags to closed/on hold/duplicate/migrated questions on question lists and review",
             "extended_description": "Adds a coloured box at the end of a title (that replaces the standard [duplicate], etc.) in question lists to more easily tell what the state of a question is",
             "meta": "https://meta.stackexchange.com/q/257021",
-            "match": "*://*.com/,*://*.*/questions*",
-            "exclude": "*://chat.*.com/*,SE1.0,*://*/review*",
+            "match": "*://*.*/,*://*.*/questions*,*://*.*/review*",
```

Comment:
Are there any SE domains that aren't `.com`?

Reply:
Yes, https://mathoverflow.net.